Marlin/pins_ZRIB_2.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#if HOTENDS > 2
+  #error "Zrib v2 supports up to 2 hotends. Comment this line to keep going."
+#endif
+
+#define BOARD_NAME "ZRIB v2"
+
+#include "pins_RAMPS.h"
+
+#undef FAN1_PIN
+#define FAN1_PIN 6
```

Comment:
RAMPS accepts some variants, so this MOSFET doesn't necessarily have to be for FAN1. To allow RAMPS to be more flexible in its pin choices, you may want to instead put this line above the `#include` instead:
```cpp
#define RAMPS_D8_PIN 6
```